pop-api/examples/fungibles/lib.rs

```diff
@@ -95,91 +79,34 @@ mod fungibles {
 		#[ink(message)]
 		pub fn decrease_allowance(
 			&mut self,
-			id: AssetId,
+			id: TokenId,
 			spender: AccountId,
 			value: Balance,
 		) -> Result<()> {
 			api::decrease_allowance(id, spender, value)
 		}
 
-		/// 2. PSP-22 Metadata Interface:
-		/// - token_name
-		/// - token_symbol
-		/// - token_decimals
-
 		#[ink(message)]
-		pub fn token_name(&self, id: AssetId) -> Result<Vec<u8>> {
+		pub fn token_name(&self, id: TokenId) -> Result<Vec<u8>> {
 			api::token_name(id)
 		}
 
 		#[ink(message)]
-		pub fn token_symbol(&self, id: AssetId) -> Result<Vec<u8>> {
+		pub fn token_symbol(&self, id: TokenId) -> Result<Vec<u8>> {
 			api::token_symbol(id)
 		}
 
 		#[ink(message)]
-		pub fn token_decimals(&self, id: AssetId) -> Result<u8> {
+		pub fn token_decimals(&self, id: TokenId) -> Result<u8> {
 			api::token_decimals(id)
 		}
-
-		// 3. Asset Management:
-		// - create
-		// - start_destroy
-		// - destroy_accounts
-		// - destroy_approvals
-		// - finish_destroy
-		// - set_metadata
-		// - clear_metadata
-
-		// #[ink(message)]
-		// pub fn create(&self, id: AssetId, admin: AccountId, min_balance: Balance) -> Result<()> {
-		// 	ink::env::debug_println!(
-		// 		"PopApiFungiblesExample::create: id: {:?} admin: {:?} min_balance: {:?}",
-		// 		id,
-		// 		admin,
-		// 		min_balance,
-		// 	);
-		// 	let result = api::create(id, admin, min_balance);
-		// 	ink::env::debug_println!("Result: {:?}", result);
-		// result.map_err(|e| e.into())
-		// result
-		// }
-
-		// #[ink(message)]
-		// pub fn set_metadata(
-		// 	&self,
-		// 	id: AssetId,
-		// 	name: Vec<u8>,
-		// 	symbol: Vec<u8>,
-		// 	decimals: u8,
-		// ) -> Result<()> {
-		// 	ink::env::debug_println!(
-		// 		"PopApiFungiblesExample::set_metadata: id: {:?} name: {:?} symbol: {:?}, decimals: {:?}",
-		// 		id,
-		// 		name,
-		// 		symbol,
-		// 		decimals,
-		// 	);
-		// 	let result = api::set_metadata(id, name, symbol, decimals);
-		// 	ink::env::debug_println!("Result: {:?}", result);
-		// 	// result.map_err(|e| e.into())
-		// 	result
-		// }
-		//
-		// #[ink(message)]
-		// pub fn asset_exists(&self, id: AssetId) -> Result<bool> {
-		// 	// api::asset_exists(id).map_err(|e| e.into())
-		// 	api::asset_exists(id)
-		// }
 	}
 
 	#[cfg(test)]
 	mod tests {
 		use super::*;
 
 		#[ink::test]
-		fn default_works() {
-			PopApiFungiblesExample::new();
-		}
+		fn default_works() {}
```

Comment:
Maybe we should just remove the test module. FYI, this is replaced with `drink` test in this PR: https://github.com/r0gue-io/pop-node/pull/277. Hence, does not matter wether we remove it or not.

Reply:
Will keep it for now, can be removed it with your PR